package.json

```diff
@@ -37,7 +37,8 @@
     "test": "yarn workspaces foreach -v run test",
     "test:build:verify": "yarn workspaces foreach -v run test:build:verify",
     "test:e2e": "yarn workspaces foreach -v run test:e2e",
-    "test:debug": "DEBUG=true yarn workspaces foreach -v run test"
+    "test:debug": "DEBUG=true yarn workspaces foreach -v run test",
+    "tx-cbor": "tsx packages/cardano-services/scripts/tx-cbor.js"
```

Comment:
Would be nice to add a paragraph in README about how to use this script. Also, not sure about the location of the script: we do have `scripts/` in monorepo root. Maybe it would be better to hoist it there?

Reply:
> Would be nice to add a paragraph in README about how to use this script.

Going to add it.

> Also, not sure about the location of the script: we do have scripts/ in monorepo root. Maybe it would be better to hoist it there?

That was my fist attempt, but I got some import issues (I don't remember the details) so I chosen this option as fallback to save some time.

Reply:
7347e1b7ab8 should solve this